Examples/uiTextBoxEx-WordControl/src/uiTextBoxEx.cls

```diff
@@ -55,13 +55,17 @@ Friend Sub protInit(fr As MSForms.Frame)
   
   'Set bindings to word
   Set pAppWord = CreateObject("Word.Application")
-  pAppWord.Visible = True
+  pAppWord.Visible = False
```

Comment:
A little confusing to me, why have you changed this to false? Is this a mistake?

Reply:
```suggestion
  pAppWord.Visible = true
```

Reply:
It is a mistake and it isn't - I had originally changed it because I had misread your coment ("Sometimes ui appears to go offscreen") as meaning you coudl see the Word app in it normal state on startup, but when I ran the code I only ever saw the textbox appear (whether set to true or false), so on balance I decided not to make the change, but I see that's not the version I committed. Apologies.